kafka/admin/client.py

```diff
@@ -738,7 +767,12 @@ def list_consumer_group_offsets(self, group_id, group_coordinator_id=None,
                     topics_partitions_dict[topic].add(partition)
                 topics_partitions = list(six.iteritems(topics_partitions_dict))
             request = OffsetFetchRequest[version](group_id, topics_partitions)
-            response = self._send_request_to_node(group_coordinator_id, request)
+            future = self._send_request_to_node(group_coordinator_id, request)
+
+            self.wait_for_futures([future])
+
+            response = future.value
+
```

Comment:
I would probably delete a lot of these blank lines, they make the code more difficult to read since IMHO they don't really separate out distinct sections of code... so it makes the code slightly less skimmable...